Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: smooth host movement cleanup

### Description (*)
This PR addresses a jitter noticed on the host when playing with lower-end devices that suffer from lower framerates.

The root cause of this issue was simply script execution order. This is outlined inside of UIStateDisplayHandler.cs, as the LateUpdate method just needs to fire after the camera has been smoothed by Cinemachine. This way, any usage of Camera.WorldToScreenPoint(..) will be true to the Camera's last updated position, removing any UI jitter.

So the execution order of this script has been set to 300, something greater than the two Cinemachine scripts (100 & 200).

Included as well is some cleanup from the previous hierarchical changes (unused references/methods).


### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-1689](https://jira.unity3d.com/browse/MTT-1689)
### Manual testing scenarios

1. Build BossRoom on a lower-end device that ideally only reaches 30 fps.
2. Play and host and notice the non-jittery UI names/healthbars.
### Questions or comments

### Contribution checklist
